test/web-platform-tests/to-run.yaml

```diff
@@ -261,6 +272,7 @@ DIR: fetch/api/headers
 header-values-normalize.any.html: [fail, fetch is not defined]
 header-values.any.html: [fail, fetch is not defined]
 headers-no-cors.any.html: [fail, Request is not defined]
+headers-normalize.any.html: [fail, Unknown]
```

Comment:
This is caused by web-platform-tests/wpt#30979, where the Deno folks noticed that you shouldn't use `String.prototype.trim()` for normalizing header values. Of course, [we made the same mistake in jsdom](https://github.com/jsdom/jsdom/blob/master/lib/jsdom/living/fetch/Headers-impl.js#L64)... 😅 

It's quite low-hanging fruit, so I'll fix it as part of this PR.